flake.nix

```diff
@@ -68,14 +97,32 @@
       let
         pkgs = import nixpkgs {
           inherit system;
-          overlays = (overlays system);
+          overlays = [ (final: prev: { llvm-backend-release = false; }) ]
+            ++ allOverlays;
         };
       in {
-        inherit overlays;
+
         packages = {
           inherit (pkgs) k;
+          pyk = pkgs.python38Packages.pyk;
+
+          update-maven = pkgs.writeShellScriptBin "update-maven" ''
+            #!/bin/sh
+            ${pkgs.nix}/bin/nix-build --no-out-link -E 'import ./nix/flake-compat-k-unwrapped.nix' \
+              || echo "^~~~ expected error"
```

Comment:
What's the error here, and why do we expect it to happen?

Reply:
this is hackery I inherited :P here is the comment from the original code, perhaps it should be copied over to the flake
https://github.com/runtimeverification/k/blob/aeecef0c3de3f13d3c33fb42e77cef5ef4e6f940/nix/update-maven.sh#L7-L10

Reply:
added a comment in place